bigtable/client/filters.h

```diff
@@ -249,6 +381,23 @@ class Filter {
     return tmp;
   }
 
+  /**
+   * Returns a filter that applies a label to each value.
+   *
+   * Each value accepted by previous filters in modified to include the @p
+   * label.
+   *
+   * @note Currently it is not possible to apply more than one label in a
+   *     filter expression, that is, a chain can only contain a single
+   *     ApplyLabelTransformer() filter.  This limitation may be lifted in
+   *     the future.  It is possible to have multiple ApplyLabelTransformer
+   *     filters in a Union() filter, though in this case each copy of a cell
+   *     gets a different label.
+   *
+   * @param label the label applied to each cell.  The labels must be at most 15
+   *     characters long, and must match the `[a-z0-9\\-]` pattern.
+   * TODO(#84) - change this if we decide to validate inputs in the client side
```

Comment:
Will this TODO be automatically included in the documentation on the previous line for `@param label`? Did you want that to happen? Do we need a blank line between the arg doc comment and the TODO to dissociate them?

Same comment elsewhere, I've seen this pattern a few times.

Reply:
Did not want that, thanks for catching it.  Fixed, and looked for the pattern elsewhere too.